File: app/src/main/java/net/simplifiedcoding/simplegame/MainActivity.java
package net.simplifiedcoding.simplegame;

import android.content.Intent;
import android.content.pm.ActivityInfo;
import android.media.Image;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageButton;

public class MainActivity extends AppCompatActivity implements View.OnClickListener {

    private ImageButton buttonPlay;
    private ImageButton buttonScore;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE);
        buttonPlay = (ImageButton) findViewById(R.id.buttonPlay);
        buttonPlay.setOnClickListener(this);
        buttonScore = (ImageButton) findViewById(R.id.buttonScore);
        buttonScore.setOnClickListener(this);
    }

    @Override
    public void onClick(View v) {

        startActivity(new Intent(this, GameActivity.class));
    }

}

File: app/src/main/java/net/simplifiedcoding/simplegame/MainActivity.kt
package net.simplifiedcoding.simplegame

import android.content.Intent
import android.content.pm.ActivityInfo
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.View
import android.widget.ImageButton

class MainActivity : AppCompatActivity(), View.OnClickListener {
    private var buttonPlay: ImageButton? = null
    private var buttonScore: ImageButton? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        //requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE
        buttonPlay = findViewById<View>(R.id.buttonPlay) as ImageButton
        buttonPlay!!.setOnClickListener(this)
        buttonScore = findViewById<View>(R.id.buttonScore) as ImageButton
        buttonScore!!.setOnClickListener(this)
    }

    override fun onClick(v: View) {
        startActivity(Intent(this, GameActivity::class.java))
    }
}